targets/csharp-unity-gameserver/make.js

```diff
@@ -0,0 +1,419 @@
+var path = require("path");
+
+exports.putInRoot = true;
+exports.makeServerAPI = function (apis, sourceDir, apiOutputDir) {
+    var gameServerApis = [];
+    for(var i=0; i<apis.length; i++){
+        var api = apis[i];
+        if(apis.name != "Admin"){
```

Comment:
It would be nicer long term to filter positively instead of negatively.
If === "Server" and if === "Matchmaker".